### ISSUE TITLE
decK sync fails with Kong EE Free version

### ISSUE THREAD

**Issue by vvidovic** [Kong/deck#357]
decK calls `GET /mtls-auths` since version 1.2.0.

Kong OSS returns 404 for that endpoint while Kong EE Free returns 403 for the same endpoint, resulting in an error when executing the `deck sync` command.

Here is the example of the `deck sync` call:
```
$ deck.1.6.0 sync
Error: mtls-auths: HTTP status 403 (message: "Forbidden")
mtls-auths: HTTP status 403 (message: "Forbidden")
```

**Comment by vvidovic** [comment]
I just pulled the latest version from GitHub and this error doesn't occur. Just to be sure, I checked out tag v1.6.0 from GitHub and an error occurs (as expected).

This issue is already fixed by https://github.com/Kong/deck/pull/321 and will be included in the next release.

Probably this issue should be closed as soon as the new version is released.

**Comment by hbagdi** [comment]
This is duplicate of https://github.com/Kong/deck/issues/274.

**Comment by ajeetshah** [comment]
I found it to be fixed in deck version 1.7.0 :)
